grep/grep_task.py

```diff
@@ -0,0 +1,92 @@
+# Task from https://exercism.org/tracks/python/exercises/grep
+
+def grep(pattern, flags, files):
+    if "-i" in flags:
+        pattern = pattern.lower()
+    result, non_matching_lines = "", ""
+
+    if len(files) == 1:
+        datafile = get_data_from_file(files[0])
+        line_counter = 1
+        for line in datafile:
+            result, non_matching_lines = get_matching_lines_single_file(line_counter, line, pattern, result, flags,
+                                                                        non_matching_lines)
+            line_counter += 1
+    elif len(files) > 1:
+        for i in range(len(files)):
+            datafile = get_data_from_file(files[i])
+            line_counter = 1
+            for line in datafile:
+                line_to_match = line
+                if "-i" in flags:
```

Comment:
Remove hardcoded values here. Make a separate constant outside with all this values.